src/android/ContentFilesystem.java

```diff
@@ -18,22 +18,19 @@ Licensed to the Apache Software Foundation (ASF) under one
  */
 package org.apache.cordova.file;
 
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.OutputStream;
-
-import org.apache.cordova.CordovaResourceApi;
-import org.json.JSONArray;
-import org.json.JSONException;
-import org.json.JSONObject;
-
 import android.content.ContentResolver;
 import android.content.Context;
 import android.database.Cursor;
 import android.net.Uri;
+import android.provider.DocumentsContract;
 import android.provider.MediaStore;
 import android.provider.OpenableColumns;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import org.apache.cordova.CordovaResourceApi;
+import org.json.JSONException;
+import org.json.JSONObject;
```

Comment:
LULZ! They follow the "Android Studio/Eclipse put it there for me style".

Seriously, Cordova, Android and Java imports should be grouped separately, but I'm not completely concerned about the order of the groupings.